packages/probing-frontend/src/pages/MonitoringDetailPage/components/MonitoringEserviceDetail.tsx

```diff
@@ -0,0 +1,52 @@
+import type { MainEservice } from '@/api/monitoring/monitoring.models'
+import { Box } from '@mui/material'
+import { ButtonNaked } from '@pagopa/mui-italia'
+import { useTranslation } from 'react-i18next'
+import LaunchIcon from '@mui/icons-material/Launch'
+import LockIcon from '@mui/icons-material/Lock'
+import { Link } from 'react-router-dom'
+import { MonitoringInformationContainer } from './MonitoringInformationContainer'
+import { CATALOGUE_BASE_PATH } from '@/config/constants'
+
+export const MonitoringEserviceDetail = ({
+  eservicesDetail,
+}: {
+  eservicesDetail: MainEservice | undefined
+}) => {
+  const { t } = useTranslation('common', {
+    keyPrefix: 'detailsPage',
+  })
+
+  const getUrl = () => {
+    return `${CATALOGUE_BASE_PATH}ui/it/fruizione/catalogo-e-service/${eservicesDetail?.eserviceId}/${eservicesDetail?.versionId}`
+  }
+
+  return (
+    <Box sx={{ width: '100%', maxWidth: '600px' }}>
+      <MonitoringInformationContainer
+        label="producerName"
+        content={eservicesDetail?.producerName || ''}
+      />
+      <MonitoringInformationContainer
+        label="version"
+        content={eservicesDetail?.versionNumber || ''}
+      />
+      <MonitoringInformationContainer
+        label="eServiceTab"
+        content={
+          <ButtonNaked
+            color="primary"
+            size="small"
+            startIcon={<LockIcon />}
+            endIcon={<LaunchIcon />}
+            component={Link}
+            target="_blank"
+            to={getUrl()}
+          >
+            {t('viewInCatalog')}
+          </ButtonNaked>
```

Comment:
Here the router Link should be used. I tried it locally and I discovered a bug where the `target` prop was not being accepted by the props typing. I solved this issue in [this PR](https://github.com/pagopa/interop-fe-commons/pull/47) in the commons repo.

Please update this part con the router link, put the `target` prop and feel free to `ts-ignore` the typescript error

Reply:
Can't do that because the function `getUrl()` returns a string and not a key of permitted routes, since it's an external route

Reply:
Thanks for pointing it out to me. I would then rename the `getUrl` function to something more descriptive, like `getExternalCatalogUrl`